mathesar/database/base.py

```diff
@@ -1,33 +1,12 @@
 from django.conf import settings
-from sqlalchemy import MetaData, create_engine, inspect
-
-from mathesar.settings import mathesar_settings
-from mathesar.database import types
-
-APP_PREFIX = mathesar_settings["APP_PREFIX"]
-ID = f"{APP_PREFIX}id"
-
-
-def create_engine_with_custom_types(*args, **kwargs):
-    engine = create_engine(*args, **kwargs)
-    # We need to add our custom types to any engine created for SQLALchemy use
-    # so that they can be used for reflection
-    engine.dialect.ischema_names.update(types.CUSTOM_TYPE_DICT)
-    return engine
+from mathesar_db import engine
```

Comment:
Could we just name this `db` or `database`? It seems redundant to call it `mathesar_db` when it's inside the `mathesar` project.

I think we should also rename `mathesar` the Django app to `web`, but I can do that when I'm renaming collections, etc.

Reply:
I was trying to avoid confusion with `mathesar.database` .  I'll change it to `db` for the moment.